.github/workflows/create-release.yml

```diff
@@ -72,7 +72,51 @@ jobs:
   run-unit-tests:
     name: Unit tests
     needs: validate-release
-    uses: kyma-project/eventing-tools/.github/workflows/unit-test-reusable.yml@main
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Cache Go dependencies
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
```

Comment:
Why not `1.22`?

Reply:
The [PR jobs](https://github.com/kyma-project/eventing-auth-manager/blob/main/.github/workflows/unit-test.yml) are still using 1.21 so just used the same version for now. Later we can bump them together.